eras/conway/impl/src/Cardano/Ledger/Conway/Governance.hs

```diff
@@ -345,3 +356,7 @@ toConwayGovernancePairs cg@(ConwayGovernance _ _) =
 instance EraPParams (ConwayEra c) => EraGovernance (ConwayEra c) where
   type GovernanceState (ConwayEra c) = ConwayGovernance (ConwayEra c)
   getConstitutionHash g = Just $ g ^. cgRatifyL . rsEnactStateL . ensConstitutionL
+
+  gsPpL = cgRatifyL . rsEnactStateL . ensPParamsL
+
+  gsPrevPpL = cgRatifyL . rsEnactStateL . ensPrevPParamsL
```

Comment:
Convention is opposite with all lenses we have (eg. `valueTxOutL`). Also, let's make it a bit more verbose, otherwise there are way too many "pp"s all over the codebase :smile: 
```suggestion
  curPParamsGovStateL = cgRatifyL . rsEnactStateL . ensPParamsL

  prevPParamsGovStateL = cgRatifyL . rsEnactStateL . ensPrevPParamsL
```